install.sh

```diff
@@ -1,15 +1,5 @@
-set -e
-pip install datasets
-
-# Install langdetect
-pip install langdetect
-
-# Install the fasttext python package
-pip install git+https://github.com/facebookresearch/fastText.git
-# This version of numpy is required for fasttext models!
-pip install numpy==1.25.2
-
-pip install black
+#!/bin/bash
+set -euo pipefail
```

Comment:
This is the "unofficial bash strictmode": http://redsymbol.net/articles/unofficial-bash-strict-mode/